subcommand/inject-connect/command_test.go

```diff
@@ -272,7 +253,7 @@ func testSignalHandling(sig os.Signal) func(*testing.T) {
 		select {
 		case exitCode := <-exitChan:
 			require.Equal(t, 0, exitCode, ui.ErrorWriter.String())
-		case <-time.After(time.Second * 1):
+		case <-time.After(time.Second * 2):
```

Comment:
curious why we needed to bump this time?

Reply:
Nice catch! The manager did not exit within 1 second but once i bumped the wait time to 2 seconds, we would always exit 0. My suspicion here is that the manager just takes a little longer to exit once it receives the cancel signal.

Reply:
oh got it! thanks for clarifying!